.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Run client tests
+
+on:
+  pull_request:
+    # only runs on PRs that target main
+    branches:
+      - main
```

Comment:
We could add a path filter to just run on changes to /client
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
added